<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>402</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="add_cinema_lab">
           <property name="text">
            <string>Добавить кинотеатр</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="cinema_name_for_cinema_lab">
             <property name="text">
              <string>Название кинотеатра:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="enter_cinema_name_for_cinema"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="id_for_cinema_opt_lab">
             <property name="text">
              <string>Id *</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="enter_id_for_cinema_opt"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="add_cinema_btn">
           <property name="text">
            <string>Добавить кинотеатр</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="add_hall_lab">
           <property name="text">
            <string>Добавить зал в кинотеатр</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="cinema_id_for_hall_lab">
             <property name="text">
              <string>Id кинотеатра</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="enter_cinema_id_for_hall"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="hall_size_for_hall_lab">
             <property name="text">
              <string>Размер (в формате n1*n2)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="enter_hall_size_for_hall"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="id_for_hall_opt_lab">
             <property name="text">
              <string>Id *</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="enter_id_for_hall_opt"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="add_hall_btn">
           <property name="text">
            <string>Добавить зал</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="add_session_lab">
           <property name="text">
            <string>Добавить сеанс</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="session_hall_lab">
             <property name="text">
              <string>Id зала</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="enter_hall_id_for_session"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="session_price_lab">
             <property name="text">
              <string>Цена</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="enter_price_for_session"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_19">
           <item>
            <widget class="QPushButton" name="add_time_start_btn">
             <property name="text">
              <string>Выбрать время начала</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="enter_time_start_for_session"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="session_film_lab">
             <property name="text">
              <string>Id фильма</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="enter_film_id_for_session"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLabel" name="id_for_session_opt_lab">
             <property name="text">
              <string>Id *</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="enter_id_for_session_opt"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="add_session_btn">
           <property name="text">
            <string>Добавить сеанс</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="add_film_lab">
           <property name="text">
            <string>Добавить фильм</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="film_name_for_film_lab">
             <property name="text">
              <string>Название фильма</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="enter_film_name_for_film"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_20">
           <item>
            <widget class="QLabel" name="dur_for_film">
             <property name="text">
              <string>Продолжительность (в минутах)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="enter_dur_for_film"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_21">
           <item>
            <widget class="QPushButton" name="add_poster_btn">
             <property name="text">
              <string>Добавить афишу к фильму</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="enter_poster_for_film"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="id_for_film_opt_lab">
             <property name="text">
              <string>Id *</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="enter_id_for_film_opt"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="add_film_btn">
           <property name="text">
            <string>Добавить фильм</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>801</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_18">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="del_cinema_lab">
         <property name="text">
          <string>Удалить кинотеатр</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QLabel" name="delete_cinema_by_id_lab">
           <property name="text">
            <string>Id кинотеатра</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="enter_cinema_id_for_del"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="del_cinema_btn">
         <property name="text">
          <string>Удалить кинотеатр</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="del_hall_lab">
         <property name="text">
          <string>Удалить зал</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QLabel" name="delete_hall_by_id_lab">
           <property name="text">
            <string>Id зала</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="enter_hall_id_for_del"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="del_hall_btn">
         <property name="text">
          <string>Удалить зал</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="del_seession_lab">
         <property name="text">
          <string>Удалить сеанс</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="delete_session_by_id_lab">
           <property name="text">
            <string>Id сеанса</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="enter_session_id_for_del"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="del_session_btn">
         <property name="text">
          <string>Удалить сеанс</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="del_film_lab">
         <property name="text">
          <string>Удалить фильм</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="delete_film_by_id_lab">
           <property name="text">
            <string>Id фильма</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="enter_film_id_for_del"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="del_film_btn">
         <property name="text">
          <string>Удалить фильм</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>500</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Вернуться к авторизации</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Таблицы</string>
    </property>
    <addaction name="show_cinemas"/>
    <addaction name="show_halls"/>
    <addaction name="show_sessions"/>
    <addaction name="show_films"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="show_cinemas">
   <property name="text">
    <string>Кинотеатры</string>
   </property>
  </action>
  <action name="show_halls">
   <property name="text">
    <string>Залы</string>
   </property>
  </action>
  <action name="show_sessions">
   <property name="text">
    <string>Сеансы</string>
   </property>
  </action>
  <action name="show_films">
   <property name="text">
    <string>Фильмы</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
